system-addon/content-src/components/Sections/_Sections.scss

```diff
@@ -0,0 +1,11 @@
+.sections-list {
+  padding: 0;
+
+  .section-list {
+    padding: 0;
+    img {
```

Comment:
I'm assuming this is for your dummy section, I would probably just put it inline in that component to prevent having to remove stuff from here later.

That said, adding custom CSS might be something we'll have to figure out later